src/assets/lima-config.yaml

```
@@ -30,3 +30,9 @@ provision:
     done
     echo "Done mounting Rancher persistent volumes."
     exit "${errors}"
+- # This sets up cron (used for logrotate)
+  mode: system
+  script: |
+    #!/bin/sh
+    rc-update add crond default
+    rc-service crond start
```

I was going to suggest:

```suggestion
    rc-service --ifnotstarted crond start
 ```

in case the service is already running after a reboot, but testing shows that the exit code still remains `0`, so it doesn't really matter:

```console
lima-rancher-desktop:/Users/jan# rc-service crond start; echo $?
 * WARNING: crond has already been started
0
```

---

In which case, I'll do nothing here :)